src/hooks/spoofing/write_player_gamer_data_node.cpp

```diff
@@ -16,9 +16,9 @@ namespace big
 			node->m_clan_data.m_clan_member_count = 15;
 			node->m_clan_data.m_clan_created_time = 420;
 
-			strcpy(node->m_clan_data.m_clan_tag, g.spoofing.crew_tag.c_str());
-			node->m_clan_data.m_is_system_clan = g.spoofing.rockstar_crew;
-			node->m_clan_data.m_is_clan_open = g.spoofing.square_crew_tag;
+			memcpy(node->m_clan_data.m_clan_tag, g.spoofing.crew_tag.c_str(), std::min(g.spoofing.crew_tag.size(), sizeof(node->m_clan_data.m_clan_tag)));
+        		node->m_clan_data.m_is_system_clan = g.spoofing.rockstar_crew;
+        		node->m_clan_data.m_is_clan_open = g.spoofing.square_crew_tag;
```

Comment:
Using strcpy without checking the size is fatal, but there is already a check here https://github.com/YimMenu/YimMenu/blob/86c44f38b5a480cb13138d177a64659f4aa3ac71/src/views/network/view_spoofing.cpp#L27-L29, so memcpy is not needed

Reply:
Though the code you linked to checks against the m_name member var and not the m_clan_tag member.

Reply:
just edit